src/main/java/cn/nukkit/Player.java

```diff
@@ -1005,10 +1005,10 @@ protected boolean orderChunks() {
             this.unloadChunk(Level.getHashX(index), Level.getHashZ(index));
         }
 
-        if (!loadQueue.isEmpty()) {
+        if (!loadQueue.isEmpty() && !lastChunk.isEmpty()) {
             NetworkChunkPublisherUpdatePacket packet = new NetworkChunkPublisherUpdatePacket();
             packet.position = this.asBlockVector3();
-            packet.radius = viewDistance << 4;
+            packet.radius = viewDistance * 16; //blocks, not chunks >.>
```

Comment:
Bit shifting doesn't mean it's chunks

Reply:
https://github.com/pmmp/PocketMine-MP/commit/30f5a8fac6bbbec46583cab467dff638739e3dac#diff-e7946cb9b02a4dba1a3d5e627e31b2ccR1158

I just copied. In pmmp not happening and this is the only difference.